### ISSUE TITLE
OnSuccess not called

### ISSUE THREAD

**Issue by kashalkar** [AnthonyNahas/ngx-auth-firebaseui#257]
Face the same issue as #155 with v2.7.2

### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```
Duplicate of #155 

### OS and Version?
OSX Sierra

### Versions
Angular CLI: 7.3.9
Node: 11.3.0
OS: darwin x64
Angular: 7.2.15
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.9
@angular-devkit/build-angular     0.13.9
@angular-devkit/build-optimizer   0.13.9
@angular-devkit/build-webpack     0.13.9
@angular-devkit/core              7.3.9
@angular-devkit/schematics        7.3.9
@angular/cdk                      7.3.7
@angular/cli                      7.3.9
@angular/fire                     5.1.3
@angular/flex-layout              7.0.0-beta.24
@angular/material                 7.3.7
@angular/pwa                      0.12.4
@ngtools/webpack                  7.3.9
@schematics/angular               7.3.9
@schematics/update                0.13.9
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.29.0

### Repro steps
Same as #155 

### The log given by the failure
Same as #155 

### Desired functionality
Same as #155 

### Mention any other details that might be useful
Same as #155

**Comment by kashalkar** [comment]
Hi Anthony, 

Sorry, I had to reopen this as I face the same issue with v2.7.2. OnSuccess not called.
Tried the workaround mentioned in #155 ...but that did not help either.

Thanks, Sanjay

**Comment by AnthonyNahas** [comment]
@kashalkar you may have an issue with firestore... 

Did you enabled firestore ? 
Did you check the firestore security rules ?

**Comment by AnthonyNahas** [comment]
@kashalkar any update ? I can't reproduce that issue... please feel free to reopen this ticket if you are still facing some problems with that...

**Comment by jferg-newcontext** [comment]
I'm not @kashalkar, but I'm also running into this issue.  Unless I'm missing something, it seems like onSuccess and onError are not implemented as `@Output()`s for `ngx-auth-firebaseui-providers`, only for the basic `ngx-auth-firebaseui` component.  Am I missing something?

**Comment by AnthonyNahas** [comment]
@kashalkar I will check that

**Comment by AnthonyNahas** [comment]
@kashalkar It seems to work! check the demo app.
Did you enable firestore ? are you using the autosync feature with firestore ?

**Comment by AnthonyNahas** [comment]
@jferg-newcontext thanks for pointing to the main issue! I've just added the appropriate event emitter `Output` to the `ngx-auth-firebaseui-providers` component:

```ts
@Output() onSuccess;
@Output() onError;
```

```html
<ngx-auth-firebaseui-providers (onSuccess)="printUser($event)"
                                             (onError)="printError($event)"
                                             layout="column">
</ngx-auth-firebaseui-providers>
```

this is available in v3.2.3 and is now live

thank you 🍻

**Comment by kashalkar** [comment]
Thanks a lot ! Confirmed it works as expected.
